bin/foreman-proxy-certs-generate

```diff
@@ -20,20 +20,33 @@ class ForemanProxyCertsGenerate
 
   private
 
+  # rubocop: disable Metrics/MethodLength
   def generate_answers_file
-    temp_file('foreman-proxy-certs-answer',
-              <<-EOF
+    if File.file?('/etc/foreman-installer/scenarios.d/last_scenario.yaml')
+      scenario = YAML.load_file('/etc/foreman-installer/scenarios.d/last_scenario.yaml').fetch(:answer_file)
+      organization = begin
+                       YAML.load_file("#{scenario}")['foreman']['initial_organization']
+                     rescue
+                       nil
+                     end
```

Comment:
@stbenjam for this I had it this way since rubocop was complaining, can we disable the cop for this? I agree after making rubocop happy it looks awful and should be on a single line.